### ISSUE TITLE
Alpine Android missing build-tools?

### ISSUE THREAD

**Issue by arosca-nl** [alvr/alpine-android#8]
Since the 21st update

```
Using Docker executor with image alvrme/alpine-android:android-30 ...
Using locally found image version due to if-not-present pull policy
Using docker image sha256:bfdb4e27da52668a0cd1874260fe301338a24f481d1643188b670d9c2e1e57d5 for alvrme/alpine-android:android-30 ...
...
$ ./gradlew -q assembleProdRelease
Checking the license for package Android SDK Build-Tools 30.0.2 in /opt/sdk/licenses
License for package Android SDK Build-Tools 30.0.2 accepted.
Preparing "Install Android SDK Build-Tools 30.0.2 (revision: 30.0.2)".
"Install Android SDK Build-Tools 30.0.2 (revision: 30.0.2)" ready.
Installing Android SDK Build-Tools 30.0.2 in /opt/sdk/build-tools/30.0.2
"Install Android SDK Build-Tools 30.0.2 (revision: 30.0.2)" complete.
"Install Android SDK Build-Tools 30.0.2 (revision: 30.0.2)" finished.
Checking the license for package Android SDK Platform 30 in /opt/sdk/licenses
License for package Android SDK Platform 30 accepted.
Preparing "Install Android SDK Platform 30 (revision: 3)".
"Install Android SDK Platform 30 (revision: 3)" ready.
Installing Android SDK Platform 30 in /opt/sdk/platforms/android-30
"Install Android SDK Platform 30 (revision: 3)" complete.
"Install Android SDK Platform 30 (revision: 3)" finished.
```

while before with
```
Using Docker executor with image alvrme/alpine-android:android-30 ...
Using locally found image version due to if-not-present pull policy
Using docker image sha256:2386ebeccf176ec891787989713329e3df4b32bc73b82fdb130e79f655e64945 for alvrme/alpine-android:android-30 ...
...
$ ./gradlew -q assembleProdRelease
```
Did not install anything

PS: build-tools 30.0.3 was just released

**Comment by alvr** [comment]
Hi @arosca-nl,

it seems that the CI is caching the jobs. For example, the build [for Android 10](https://github.com/alvr/alpine-android/runs/1596516431?check_suite_focus=true#step:16:46) is using the layer of Android 9.

**Comment by alvr** [comment]
Hi @arosca-nl. This should be fixed in 55ae824. There is an error using the wrong context when building the images (moby/buildkit#1368), and therefore installed the wrong build-tools package.

I also upgraded the build-tools to 30.0.3, thanks for the heads-up.

Let me know if this is still happening.
